wru 2020 update (2.0.0)

mdblocker opened the pull request:
Changed all defaults to 2020 data, and added a message to notify users of the year being used when pulling census data.

1beb commented on R/get_census_api.R:
```diff
@@ -5,8 +5,8 @@
 #' This function obtains U.S. Census data via the public API. User
 #' can specify the variables and region(s) for which to obtain data.
 #'
-#' @param data_url URL root of the API, including the question mark,
```
Verify this change to docs please. @mdblocker

mdblocker replied:
<img width="1381" alt="image" src="https://github.com/kosukeimai/wru/assets/46643241/07ca5308-a15f-4eec-8d4c-31666d894f73">
Verified, sends me to the 2020 version of the API-endpoint (screen grab for confirmation)

1beb commented on R/get_census_api_2.R:
```diff
@@ -6,8 +6,8 @@
 #' function directly.
 #'
-#' @param data_url URL root of the API, including the question mark, 
-#'  e.g., \code{"https://api.census.gov/data/2010/dec/sf1?"}.
```
Please verify.

mdblocker replied:
verified, same 2020 endpoint as the sister R script.

mdblocker commented on R/predict_race.R:
```diff
@@ -136,8 +136,8 @@
 predict_race <- function(voter.file, census.surname = TRUE, surname.only = FALSE,
-                         surname.year = 2010, census.geo, census.key = NULL, census.data = NULL, age = FALSE,
-                         sex = FALSE, year = "2010", party = NULL, retry = 3, impute.missing = TRUE,
+                         surname.year = 2020, census.geo, census.key = NULL, census.data = NULL, age = FALSE,
```
message added, surname.year removed

1beb reviewed (changes requested):
Please remove the binary files in src/ they are a remnant of your local build that should not be distributed with the package.

mdblocker commented:
@beb I rescued the afore mentioned files from the trash and added them to .gitignore instead, added one I missed. I think I have all the files dated to my commit. Unless you see one I missed I believe this is ready to go.

1beb commented:
@mdblocker Now that we've updated the defaults we need to update the tests to reflect the updates. What I would like to do is have two separate copies. One copy with the old default arg set and a new set with the updated arg that runs off of a different census data set. You'll need to recreate the data for 2020 in tests/data/census_block_nj_2010 and save it as a new file 2020.
